src/pages/release-notes.md

```diff
@@ -42,9 +48,9 @@ By default, the versions listed in the [Tractus-X changelog](https://eclipse-tra
 
 #### Compatible versions
 
-| Component                                    | Versions                                       |
-|----------------------------------------------|------------------------------------------------|
-| Eclipse Data Space Connector (Tractus-X EDC) | [0.7.3](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.3) [Recommended] <br /> [0.7.4](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.4) <br />   [0.7.5](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.5) [Latest] |
+| Component                                    | Versions                                                                                                                                                                                                                                                                      | Comments                                                                                                                     |
+| -------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| Eclipse Data Space Connector <br /> (Tractus-X EDC) | [0.7.3](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.3) <br /> [0.7.4](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.4) <br />   [0.7.5](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.5) <br />   [0.7.6](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.6) [Latest] | Versions ≤ 0.7.5 are effected by a security issue details [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1584). |
```

Comment:
Der Link auf den PR in TX - bleibt der stabil? Gibts sonst eine andere source of truth? Wenn nicht, dann vorerst lassen, aber im Hinterkopf behalten, dass der Link ggf. obsolet wird.

Reply:
Hey - ja die bleiben stabil - und sind persitent